etc/kayobe/kolla.yml

```diff
@@ -73,7 +73,7 @@ kolla_docker_namespace: stackhpc
 #kolla_docker_registry_password:
 
 # Kolla OpenStack release version. This should be a Docker image tag.
-kolla_openstack_release: 7.0.2.6-2
+kolla_openstack_release: 7.0.2.7-1
```

Comment:
I can't see this tag in the StackHPC kolla repo (7.0.2.7). If you are just rebuilding the image with the config change below, then you can bump it to `7.0.2.6-3`.